snapcraft/parts/plugins/matter_sdk_plugin.py

```diff
@@ -111,47 +111,48 @@ def get_build_snaps(self) -> Set[str]:
 
     @overrides
     def get_build_commands(self) -> List[str]:
-        options = cast(MatterPluginProperties, self._options)
+        options = cast(MatterSdkPluginProperties, self._options)
         commands = []
 
         if self.snap_arch == "arm64":
             commands.extend(
                 [
-                    f"wget --no-verbose https://github.com/project-chip/zap/releases/download/"
-                    f"{options.matter_zap_version}/zap-linux-{self.snap_arch}.zip",
+                    "wget --no-verbose https://github.com/project-chip/zap/releases/download/"
```

Comment:
Move `https://github.com/project-chip/zap` to a constant on top, see next comment.